tests/unit/s2n_libcrypto_test.c

```diff
@@ -61,5 +62,10 @@ int main(int argc, char** argv)
         EXPECT_FALSE(s2n_supports_custom_rand());
     }
 
+    /* We expect openssl-3.0 to support providers */
+    if (strstr(env_libcrypto, "openssl") && strstr(env_libcrypto, "3")) {
+        EXPECT_TRUE(s2n_libcrypto_supports_providers());
+    }
```

Comment:
Should `buildspec_openssl3fips.yml` set `S2N_LIBCRYPTO`?

Reply:
Yes it should D: I must have forgotten a negative CI test when I added the spec!

Here's a negative CI test now: https://us-west-2.console.aws.amazon.com/codesuite/codebuild/024603541914/projects/Openssl3fipsWIP/build/Openssl3fipsWIP%3Ac4c1670b-9b51-40a6-96e5-2328ad81a548?region=us-west-2 I ran against a change that broke s2n_build_test only for openssl-3.0-fips: https://github.com/aws/s2n-tls/commit/9319bb3b9ec8ebfe504f3ffc5d164948b4e85c04